vast/CMakeLists.txt

```diff
@@ -58,12 +58,22 @@ file(
      . \"$env_dir/bin/activate\"
      python -m pip install --upgrade pip
      python -m pip install -r \"$base_dir/requirements.txt\"
-     $<$<BOOL:<${VAST_ENABLE_ARROW}>:python -m pip install pyarrow>
+     $<$<BOOL:${VAST_ENABLE_ARROW}>:python -m pip install pyarrow>
      python \"$base_dir/integration.py\" \
       --app \"$app\" \
       --directory vast-integration-test \
+      --set \"$base_dir\"/vast_integration_suite.yaml \
```

Comment:
Can you remove this line and revert the renaming _or_ adjust the default value set in `integration.py`?